tools/testsys/src/run.rs

```diff
@@ -116,6 +132,10 @@ struct CliConfig {
     /// Add secrets to the testsys agents (`--secret awsCredentials=my-secret`)
     #[clap(long, short, parse(try_from_str = parse_key_val), number_of_values = 1)]
     secret: Vec<(String, SecretName)>,
+
+    /// The endpoint IP to reserve for the vSphere control plane VMs when creating a K8s cluster
+    #[clap(long, env = "TESTSYS_CONTROL_PLANE_ENDPOINT")]
+    pub control_plane_endpoint: Option<String>,
```

Comment:
I don't love this. I'd much rather see `Infra.toml` allow us to specify a CIDR (or CIDRs) for cluster endpoint IPs as a datacenter-level option, and for `testsys` to figure out which of those IPs are available for allocation.

Reply:
A user would then have to commit to not using those ip in other applications otherwise testsys could try to use it.

Reply:
Since we can dedicate an ip to a specific variant in `Test.toml`,
```toml
[vmware-k8s-1.22]
control-plane-ip=10.26.48.22
[vmware-k8s-1.23]
control-plane-ip=10.26.48.23
```
we will work on this feature separately.